graylog2-rest-models/pom.xml

```diff
@@ -52,5 +52,13 @@
             <groupId>joda-time</groupId>
             <artifactId>joda-time</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.ext</groupId>
+            <artifactId>jersey-bean-validation</artifactId>
```

Comment:
Is this dependency really required in graylog2-rest-models? It will also be imported into the web interface this way, which doesn't use Jersey at all (except for that hack with the Jersey 1.x UriBuilder).